src/schema/top_level_files.yml

```diff
@@ -1,32 +1,41 @@
 README:
+  required: True
   extensions:
     - md
     - rst
     - None
 CHANGES:
+  required: True
   extensions:
     - md
     - rst
     - None
 dataset_description:
+  required: True
   extensions:
     - json
 genetic_info:
+  required: False
   extensions:
     - json
 participants:
+  required: True
```

Comment:
I have got "burnt" on this one recently... apparently at least how it is worded now in the spec -- it is not required! See https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file which says "Optional: Yes" although it might have meant to be for `phenotype/`?